setup.cfg

```diff
@@ -1,15 +1,4 @@
-[bdist_wheel]
-universal = 1
-
 [flake8]
 exclude = .git,.tox
 ignore = W504
 max-line-length = 100
```

Comment:
Mainly just a question for debate, but could this be moved to `.flake8` to signal that everything has been moved to Poetry?

Reply:
I have good hope that flake will add its support to pyproject.toml, so I think we can keep things like this until the support is added.